system/config/templates/features/features.yml

```diff
@@ -38,4 +38,31 @@ registry:
       - key: :drop_vlp_document
         is_enabled: <%= ENV["DROP_VLP_DOCUMENT_IS_ENABLED"] || false %>
       - key: :invert_person_association
-        is_enabled: <%= ENV["INVERT_PERSON_ASSOCIATION_IS_ENABLED"] || false %>
\ No newline at end of file
+        is_enabled: <%= ENV["INVERT_PERSON_ASSOCIATION_IS_ENABLED"] || false %>
+  - namespace:
+    - :income
+    features:
+      - key: :earned_income_filing_threshold_2020
+        item: 12_400
+        is_enabled: true
```

Comment:
we should use environment variables here instead of hard-coded values

Reply:
@markgoho for enabling the feature or for the value ?

Reply:
enabling the feature would be one definite requirement -- as far as the values themselves, if these are subject to change between clients, they should also be environment variables

however, if we expect the single value to serve for all clients for an extended period of time, we could keep them as hard-coded values

Reply:
Just heard from Sarah B. -- she says these values are the same for DC and ME and I think we'd be okay keeping them as hard-coded values

Reply:
@Battula what do we gain from this layer of indirection? i.e. moving values from where they were initially set to the resource registry?

Reply:
@markgoho Dan doesn't want hard-coded values in the system and hence introduced RR. And, it serves as one place that stores all the site level config, if that makes sense. Thanks!

Reply:
@Battula I would recommend reverting back to having the values directly in the `types.rb` file and not put anything in the resource registry

Reply:
We want site level config in RR. Thanks for your recommendation though, @markgoho !

Reply:
I'm not seeing the value in having this in the resource registry. Make the change please.